handbook/direction/code-graph/code-insights/index.md

```diff
@@ -1,44 +1,218 @@
-# Code insights goals and roadmap
+# Code Insights Direction
 
-The Code insights team focuses on bringing code insights to life as a valuable new Sourcegraph product.
+Code Insights is an upcoming product that lets you track and understand what's in your code and how it changes over time.
 
-The team's goals and roadmap contain the features and customer signals that guide our work.
+This page outlines the vision, strategy, and goals of the Code Insights team.
 
-## Goals for FY22
+Quick links:
 
-See also our [completed goals](../../../engineering/code-graph/code-insights/goals_completed.md).
+- [Code Insights Backlog](https://github.com/orgs/sourcegraph/projects/200/) <!-- Update links with your page -->
```

Comment:
Maybe link to the All issues view? This will link to the current iteration, which could be misleading for someone looking for the _Backlog_

Reply:
Think the kanban view makes the most sense, will do